doc/api/dns.md

```diff
@@ -285,6 +285,9 @@ changes:
     When `true`, the callback receives an array of
     `{ address: '1.2.3.4', ttl: 60 }` objects rather than an array of strings,
     with the TTL expressed in seconds.
+  - `search` {boolean} Resolve the DNS using the local domain name or the search 
+    list for host-name lookup.
+    When `true`, the resolve will use the search list which can create extra dns queries.
```

Comment:
Nits:
1. `dns` -> `DNS`.
2. We wrap lines after 80 characters at most. Otherwise, there will be doc linting error.

Reply:
`the resolve will use` -> `the resolver will use`? Or `the resolve will use` -> `the hostname will be resolved with`